maat-orchestration/build.gradle

```diff
@@ -37,6 +37,8 @@ dependencies {
 	implementation "uk.gov.justice.service.laa-crime:crime-commons-spring-boot-starter-rest-client:$versions.crimeCommonsVersion"
 	implementation "uk.gov.justice.service.laa-crime:crime-commons-classes:$versions.crimeCommonsVersion"
 
+	annotationProcessor "org.springframework.boot:spring-boot-configuration-processor"
```

Comment:
Spring boot configuration processor is inherited from the crime commons rest client lib. If we use the fat jar then we can remove line 40.

Change Line 37 to :       ` implementation "uk.gov.justice.service.laa-crime:crime-commons-spring-boot-starter-rest-client:$versions.crimeCommonsVersion:all"`

and we can remove Line 40

Reply:
Good shout :)